Python/import.c

```diff
@@ -575,21 +691,29 @@ _PyImport_FixupExtensionObject(PyObject *mod, PyObject *name,
 }
 
 int
-_PyImport_FixupBuiltin(PyObject *mod, const char *name)
+_PyImport_FixupBuiltin(PyObject *mod, const char *name, PyObject *modules)
 {
     int res;
     PyObject *nameobj;
     nameobj = PyUnicode_InternFromString(name);
     if (nameobj == NULL)
         return -1;
-    res = _PyImport_FixupExtensionObject(mod, nameobj, nameobj);
+    res = _PyImport_FixupExtensionObjectEx(mod, nameobj, nameobj, modules);
     Py_DECREF(nameobj);
     return res;
 }
 
 PyObject *
 _PyImport_FindExtensionObject(PyObject *name, PyObject *filename)
```

Comment:
I dislike keeping N versions of the same function. It's a private function (_Py prefix), I would prefer to change it's change prototype instead of keeping _PyImport_FindExtensionObject() unchanged and add _PyImport_FindExtensionObjectEx().

As you did for _PyImport_FindBuiltin().

Reply:
Sounds good.